prometheus_all/input.tf

```diff
@@ -18,17 +18,28 @@ variable grafana_extra_datasources { default = [] }
 variable grafana_google_jwt { default = "" }
 variable grafana_runtime_version { default = "6.5.1" }
 
+variable grafana_elasticsearch_credentials {
+  type = map
+
+  default = {
+    url      = ""
+    username = ""
+    password = ""
+  }
+}
+
 variable prometheus_memory { default = null }
 variable prometheus_disk_quota { default = null }
 variable prometheus_extra_scrape_config { default = "" }
 
 variable influxdb_service_plan { default = "tiny-1_x" }
 
-variable redis_service_instance_id { default = null }
+variable redis_services { default = [] }
 
 variable enabled_modules {
   type = list
   default = [
+    "prometheus_pushgateway",
```

Comment:
is `pushgateway` for pushing custom metrics from app code?
should it be opt-in?

Reply:
yes, I havent really finished this, so I was going to just default it for my testing, but I am worried about it having an external interface and allowing the world to push metrics in. so still needs a few changes.

Reply:
Push gateway removed